File: HTTP_Server_CLI/scenarios/2.1.1-nested_6.lua
-- @label: 6 nested routes (2 groups)
-- @group: Nested routes
-- @competitors: all
-- Round-robins through admin and account group routes.

local paths = {
   "/admin/dashboard", "/admin/settings", "/admin/users",
   "/account/profile", "/account/billing", "/account/security"
}
local counter = 0

request = function()
   counter = counter + 1
   return wrk.format("GET", paths[(counter % #paths) + 1])
end

File: HTTP_Server_CLI/scenarios/php/2.1.1-nested_6.php
<?php
// @label: 6 nested routes (2 groups)
// @group: Nested routes
// @competitors: all
// Round-robins through admin and account group routes.

return [
   'method' => 'GET',
   'paths'  => [
      '/admin/dashboard', '/admin/settings', '/admin/users',
      '/account/profile', '/account/billing', '/account/security',
   ],
];
